anvil/tests/it/otterscan_api.rs

```diff
@@ -0,0 +1,44 @@
+//! tests for otterscan endpoints
+use crate::abi::MulticallContract;
+use anvil::{spawn, NodeConfig};
+use ethers::{
+    prelude::{Middleware, SignerMiddleware},
+    signers::Signer,
+    types::BlockNumber,
+    utils::get_contract_address,
+};
+use std::sync::Arc;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn can_call_otterscan_has_code() {
```

Comment:
yep for tests this looks good. I'd say most endpoints can be tested in similar fashion to the tests in `anvil_api.rs` and `api.rs` on the `tests` folder on anvil.